app/controllers/assessments_controller.rb

```diff
@@ -111,12 +111,13 @@ def install_assessment
                                          filename)) || (filename == "..") || (filename == ".")
 
       # assessment names must be only lowercase letters and digits
-      if filename =~ /[^a-z0-9]/
+      if filename !~ /^[A-Za-z_][A-Za-z][A-Za-z0-9_-]*$/
```

Comment:
![Screenshot 2023-10-29 at 21 19 12](https://github.com/autolab/Autolab/assets/9074856/b2c2cc7c-4e17-45a6-b7b3-a6a055cd2bf7)

Regex is disallowing names that should be valid